README.md

```diff
@@ -82,7 +94,9 @@ Describe the differences between unit and functional testing. What type of testi
 Your answer:
 ```text
 
-Replace this with your answer
+Unit testing checks whether a specific method has a specific output.
+Functional testing checks whether a program as a specific functionality, which may involve multiple methods.
+RSpec is unit testing because it tests the specific methods of specific objects
```

Comment:
functional testing is testing at the level of user interaction, like clicking on links, buttons, testing admin privileges, etc..